sp800-63b/sec5_authenticators.md

```diff
@@ -31,23 +33,23 @@ Memorized secrets that are randomly chosen by the CSP (e.g., at enrollment) or b
 
 Memorized secret verifiers SHALL NOT permit the subscriber to store a "hint" that is accessible to an unauthenticated claimant. Verifiers also SHALL NOT prompt subscribers to use specific types of information (e.g., "What was the name of your first pet?") when choosing memorized secrets.
 
-When processing requests to establish and change memorized secrets, verifiers SHALL compare the prospective secrets against a list of known commonly-used, expected, and/or compromised values. For example, the list MAY include (but is not limited to):
+When processing requests to establish and change memorized secrets, verifiers SHALL compare the prospective secrets against a list that contains values known to be commonly-used, expected, or compromised. For example, the list MAY include (but is not limited to):
 
 * Passwords obtained from previous breach corpuses
```

Comment:
I'm okay with it as it is but if we get any pushback we can add some legalistic language.